ci/grunt.yml

```diff
@@ -0,0 +1,28 @@
+name: Grunt
```

Comment:
A bit more descriptive name like "NodeJs with Grunt" might be better.

Reply:
agreed. updated